docs/reference/release-notes/highlights.asciidoc

```diff
@@ -24,18 +24,154 @@ Other versions:
+some aggregations but it is corrected with the real size after the reduce
+completes. If the reduce is successful, we update the circuit breaker to remove
+the size of the source aggregations and replace the estimation with the
+serialized size of the newly reduced result.
+
+[discrete]
+[[eql-case-sensitivity-operator]]
+=== EQL: Case-sensitivity and the `:` operator
 
-// Omit the notable highlights tag for entries that only need to appear in the ES ref:
-// [discrete]
-// === Heading
-//
-// Description.
+In 7.10, we made most EQL operators and functions case-sensitive by default.
+We've also added `:`, a new case-insensitive equal operator. Designed for
+security use cases, you can use the `:` operator to search for strings in
+Windows event logs and other event data containing a mix of letter cases.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search
+{
+  "query": """
+    process where process.executable : "c:\\windows\\system32\\cmd.exe"
+  """
+}
+----
+// TEST[setup:sec_logs]
+
+For more information, see the {ref}/eql-syntax-comparison-operators.html[EQL
+syntax documentation].
+
+[discrete]
+=== Deprecating REST API access to system indices
+
+We are deprecating REST API access to system indices. Most REST API requests
+that attempt to access system indices will return the following deprecation
+warning:
+
+[source,text]
+----
+this request accesses system indices: [.system_index_name], but in a future major version, direct access to system indices will be prevented by default
+----
+
+To allow projects external to {es} additional time to adjust their access
+patterns, these warnings are currently enabled for snapshot builds only.
+
+The following REST API endpoints access system indices as part of their
+implementation and will not return the deprecation warning:
+
+* `GET _cluster/health`
+* `GET {index}/_recovery`
+* `GET _cluster/allocation/explain`
+* `GET _cluster/state`
+* `POST _cluster/reroute`
+* `GET {index}/_stats`
+* `GET {index}/_segments`
+* `GET {index}/_shard_stores`
+* `GET _cat/[indices,aliases,health,recovery,shards,segments]`
+
+[discrete]
+[[add-system-read-thread-pool]]
+=== New thread pools for system indices
+
+We've added two new thread pools for system indices: `system_read` and
+`system_write`. `system_read` is a `fixed` thread pool used to manage memory for
+read operations performed by system indices. Similarly, `system_write` is a
+`fixed` thread pool used to manage memory for write operations performed by
+system indices. Both have a maximum number of threads equal to `5`
+or half of the available processors, whichever is smaller.
+
+[discrete]
+[[metadata-flag-system-indices]]
+=== New metadata flag for system indices
+
+To prevent performance issues related to searches that accidentally target
+system indices, we've added a new metadata flag to mark system and non-system
+indices. {es} will automatically add this flag to an existing system indices
+during upgrade.
```

Comment:
@jaymode I made some changes here. Do you mind reviewing to ensure everything looks good to you?